pyscf/md/__init__.py

```diff
@@ -45,4 +45,4 @@ def set_seed(seed):
 from pyscf.md import integrators, distributions
 
 NVE = integrators.VelocityVerlet
-
+NVT_Berendson = integrators.NVTBerendson
```

Comment:
I would just remove this from the `__init__.py` file.

Reply:
Done!